.github/workflows/docker-image.yaml

```diff
@@ -0,0 +1,66 @@
+name: Docker image
+on:
+  push:
+    branches:
+      - 'develop'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build:
+    name: Build & push docker image
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+        fail-fast: false
+        matrix:
+          psql: [12,13,14,15,16]
+          postgis: [3.4]
+          os: [ubuntu-latest]
+
+    env:
+      IMG_NAME: ${{ github.repository }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.0.0
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          # list of Docker images to use as base name for tags
+          images: mobilitydb/mobilitydb
+          # generate Docker tags based on the following events/attributes
+          flavor: |
+            latest=${{ matrix.psql == '16' && github.ref_type == 'tag' }}
+            prefix=${{ matrix.psql }}-${{ matrix.postgis }}-
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{major}}.{{minor}}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
>  platforms: linux/amd64,linux/arm64

Note: Currently, there is no arm64 version of the postgis/postgis docker image, 
but it is included in the roadmap and is a high priority. 
If it's essential, I can provide access to a test arm64 image.

Reply:
I believe we can make do without it for the moment.
We can add it back when the arm64 version of postgis/postgis is ready.